client/src/components/CreatePost/Form/FirstSection.js

```diff
@@ -3,13 +3,21 @@ import { Input } from "antd";
 import styled from "styled-components";
 import Head from "./Head";
 import { Section } from "../StyledModal";
+import { theme } from "constants/theme";
+const { black } = theme.colors;
```

Comment:
needs a line space between line 6 and 7

Reply:
> @omarzoghayyer I see this as soon as I start typing in the title input field :
> <img alt="Screen Shot 2020-07-11 at 8 03 41 PM" width="940" src="https://user-images.githubusercontent.com/37174253/87237993-bd40f500-c3b1-11ea-8e4e-272b8ab86ee4.png">

This shouldn't there. Ill work on it rtw.